### PR TITLE
Complete sign up page.

### PR DESCRIPTION
**Pull request by Lenahappy24**
I completed the sign up page.

### CODE REVIEW

File: lib/views/login_register/register.dart [hunk 1]
```diff
@@ -1,11 +1,162 @@
...
   Widget build(BuildContext context) {
-    return const FunPlaceholder('Register');
+    return Scaffold(
+      body: Container(
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            const SizedBox(
+              height: 0,
+            ),
```
Comment (nate-thegrate) [inline lib/views/login_register/register.dart:18]: This empty `SizedBox` can be removed.

File: lib/views/login_register/register.dart
Comment (nate-thegrate) [inline lib/views/login_register/register.dart:1]: screenshot

This looks amazing! 🤩

File: lib/main.dart [hunk 2]
```diff
@@ -28,7 +30,7 @@ class App extends StatelessWidget {
         darkTheme: darkTheme,
         themeMode: context.watch<AppTheme>().state,
         debugShowCheckedModeBanner: false,
-        home: const ChooseAnyView(),
+        home: const RegisterScreen(),
```
Comment (nate-thegrate) [inline lib/main.dart:33]: Let's change this back, and add a "register screen" button to `ChooseAnyView`

**Review by nate-thegrate — APPROVED** [review]
Thanks for making this great addition!

**Comment by Lenahappy24** [comment]
> Thanks for making this great addition!

Thank you very much!

**Pull request merged**